installer/manifests/keptn/rbac.yaml

```diff
@@ -78,6 +78,28 @@ metadata:
   labels:
     "app": "keptn"
 rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - get
```

Comment:
Can the `create` be added here because this would allow us to limit the resources?

Reply:
according to the k8s docs, `create` cannot be restricted by the `resourceName` attribute, because at the time of the creation the object is not available yet and hence gannot be verified (see https://github.com/kubernetes/kubernetes/issues/80295)

Reply:
Thanks for the explanation.